components/org.wso2.carbon.identity.oidc.session/src/test/resources/repository/conf/carbon.xml

```diff
@@ -0,0 +1,684 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Are you sure that a carbon.xml file is required ? I haven't come across any case where carbon.xml file is required

Reply:
I'm getting this error when I don't have carbon.xml file. 
32 [main] ERROR org.wso2.carbon.utils.CarbonUtils - Cannot read file /home/piraveena/Documents/identity-inbound-oauth-unit-test/identity-inbound-auth-oauth/components/org.wso2.carbon.identity.oidc.session/src/test/resources/repository/conf/carbon.xml
java.io.FileNotFoundException: /home/piraveena/Documents/identity-inbound-oauth-unit-test/identity-inbound-auth-oauth/components/org.wso2.carbon.identity.oidc.session/src/test/resources/repository/conf/carbon.xml (No such file or directory)